tests/prague/eip7702_set_code_tx/test_set_code_txs.py

```diff
@@ -37,6 +36,7 @@
     Storage,
     Transaction,
     TransactionException,
+    TransactionTestFiller,
```

Comment:
hm why this file is affected?  
I think we shall separate the transaction test fillers into separate folder because those are not converted into hive or bc format.

Reply:
Should we not have them in the bc or bc_engine format?
For the latter we can see if client correctly reject invalid txs via engine new payload

Reply:
> Should we not have them in the bc or bc_engine format? For the latter we can see if client correctly reject invalid txs via engine new payload

I tried to add this but we need to update some error messages that we receive from the execution-specs, so we can leave this out for a follow-up PR.